SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -110,6 +111,28 @@ bool USpatialNetDriver::InitBase(bool bInitAsClient, FNetworkNotify* InNotify, c
 		return false;
 	}
 
+#if WITH_EDITOR
+	// If we're launching in PIE then ensure there is a deployment running before connecting.
+	FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
+	FLocalDeploymentManager* LocalDeploymentManager = GDKServices.GetLocalDeploymentManager();
+
+	// Just connect if a deployment is running.
+	if (!LocalDeploymentManager->IsLocalDeploymentRunning() || LocalDeploymentManager->IsDeploymentStopping() || LocalDeploymentManager->IsDeploymentStarting())
+	{
+		UE_LOG(LogSpatialOSNetDriver, Log, TEXT("Waiting for local spatial depoyment to start before connecting..."));
+		SpatialDeploymentStartHandle = LocalDeploymentManager->OnDeploymentStart.AddLambda([this, URL]
+		{
+			UE_LOG(LogSpatialOSNetDriver, Log, TEXT("Local deployment started, connecting with URL: %s"), *URL.ToString());
+			InitiateConnectionToSpatialOS(URL);
+
+			FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
```

Comment:
can this be const?

```suggestion
			const FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
```

Reply:
It seems that when getting modules like this you cannot use const. Checked in Unreal code too and they never use const. Attempting to use it results in compile errors.